lib/miq_unicode.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module UnicodeString
+    refine String do
+      def UnicodeToUtf8
+        dup.force_encoding("UTF-16LE").encode("UTF-8")
+      end
+
+      def UnicodeToUtf8!
+        force_encoding("UTF-16LE").encode!("UTF-8")
+      end
+
+      def Utf8ToUnicode
+        dup.force_encoding("UTF-8").encode("UTF-16LE")
+      end
+
+      def Utf8ToUnicode!
+        force_encoding("UTF-8").encode!("UTF-16LE")
+      end
+
+      def AsciiToUtf8
+        dup.force_encoding("ISO-8859-1").encode("UTF-8")
```

Comment:
should this be simplified to `dup.AsciiToUtf8!`?

Reply:
Done.